store/sqlstore/upgrade.go

```diff
@@ -1055,99 +1055,10 @@ func upgradeDatabaseToVersion535(sqlStore *SqlStore) {
 		sqlStore.CreateUniqueCompositeIndexIfNotExists(RemoteClusterSiteURLUniqueIndex, "RemoteClusters", uniquenessColumns)
 		sqlStore.CreateColumnIfNotExists("SharedChannelUsers", "ChannelId", "VARCHAR(26)", "VARCHAR(26)", "")
 
-		rootCountMigration(sqlStore)
+		if err := sqlStore.Thread().RootCountMigration(); err != nil {
+			mlog.Error("Error performing RootCountMigration", mlog.Err(err))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.